consensus/core/src/round_prober.rs

```diff
@@ -266,9 +373,8 @@ fn compute_quorum_round(
     let mut total_stake = 0;
     let mut low = 0;
     for (round, stake) in rounds_with_stake.iter().rev() {
-        let reached_quorum_before = total_stake >= committee.quorum_threshold();
```

Comment:
@mwtian can you just double check I am good to remove this, when I reviewed your PR initially I convinced myself it was needed but now I have convinced myself it is not.

Reply:
Probably a leftover from a previous impl. This shouldn't be needed anymore.